plasmapy/formulary/collisions.py

```diff
@@ -2214,34 +2305,51 @@ def __init__(
         self.test_particle = test_particle
         self.field_particle = field_particle
         self.v_a = v_a
-        self.T_a = (
-            T_a
-            if T_a is not None
-            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
-        )
-        self.n_b = n_b
+        self.T_a = T_a
+        self.n_a = n_a
         self.T_b = T_b
+        self.n_b = n_b
         self.Coulomb_log = (
             Coulomb_log
             if isinstance(Coulomb_log, u.Quantity)
             else Coulomb_log * u.dimensionless_unscaled
         )
 
+    @cached_property
+    def _v_T_a(self):
+        """
+        The thermal velocity of the test particle species.
+        """
+
+        return (
```

Comment:
I don't think you should have this if/else statement, because there can be a v_T_a even if a separate v_a is not set. 

If the test particles have a drift velocity, then their velocity is 

v_a = v_drift + v_T_a

if there is no drift, then 

v_a = v_T_a

What is confusing is that if the thermal motion is small compared to the drift motion (e.g. v_T_a << v_drift) for example in the fast limit of the original four frequencies we have 

v_a = v_drift 

So I think our input v_a from the user should really be interpreted as v_drift. If no T_a is set, then we should assume it's small and v_a = v_drift. If both v_drift and T_a are given, we compute v_T_a and add the velocities together to get v_a. And if only T_a is given, we should set v_a = v_T_a.  Does that make sense?